src/core/mac/mac_links.hpp

```diff
@@ -461,28 +461,22 @@ class Links : public InstanceLocator
 
 #if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
     /**
-     * Configures CSL parameters in all radios.
+     * Sets CSL parameters in all radios.
      *
      * @param[in]  aPeriod    The CSL period.
      * @param[in]  aChannel   The CSL channel.
      * @param[in]  aShortAddr The short source address of CSL receiver's peer.
      * @param[in]  aExtAddr   The extended source address of CSL receiver's peer.
-     *
-     * @retval  TRUE if CSL Period or CSL Channel changed.
-     * @retval  FALSE if CSL Period and CSL Channel did not change.
      */
-    bool UpdateCsl(uint16_t aPeriod, uint8_t aChannel, ShortAddress aShortAddr, const ExtAddress &aExtAddr)
+    void SetCslParams(uint16_t aPeriod, uint8_t aChannel, otShortAddress aShortAddr, const Mac::ExtAddress &aExtAddr)
```

Comment:
Use `ShortAddress` instead of `otShortAddress`?
```suggestion
    void SetCslParams(uint16_t aPeriod, uint8_t aChannel, ShortAddress aShortAddr, const Mac::ExtAddress &aExtAddr)
```